browser/help/topics/v4-browser-hts.md

```diff
@@ -0,0 +1,223 @@
+- `coverage`: Struct containing coverage information for locus.
+  - `exome`: Struct containing exome coverage information.
+  - `mean`: Mean depth of coverage at this locus.
+  - `median`: Median depth of coverage at this locus.
+  - `over_1`: Percentage of samples with a coverage greater than 1 at this locus.
+  - `over_5`: TPercentage of samples with a coverage greater than 5 at this locus
+  - `over_10`: Percentage of samples with a coverage greater than 10 at this locus
+  - `over_15`: Percentage of samples with a coverage greater than 15 at this locus.
+  - `over_20`: Percentage of samples with a coverage greater than 20 at this locus
+  - `over_25`: Percentage of samples with a coverage greater than 25 at this locus.
+  - `over_30`: Percentage of samples with a coverage greater than 30 at this locus.
+  - `over_50`: Percentage of samples with a coverage greater than 50 at this locus.
+  - `over_100`: Percentage of samples with a coverage greater than 100 at this locus.
+  - `genome`: Struct containing genome coverage information. Contains the same fields as `exome` above.
+  - `transcript_consequences`: Array containing variant transcript consequence information.
+  - `biotype`: Transcript biotype.
+  - `consequence_terms`: Array of predicted functional consequences.
+  - `domains`: Sset containing protein domains affected by variant.
+  - `gene_id`: Unique ID of gene associated with transcript.
+  - `hgvsc`: HGVS coding sequence notation for variant.
+  - `hgvsp`: HGVS protein notation for variant.
+  - `is_canonical`: Whether transcript is the canonical transcript.
+  - `lof_filter`: Variant LoF filters (from [LOFTEE](https://github.com/konradjk/loftee)).
+  - `lof_flags`: LOFTEE flags.
+  - `lof`: Variant LOFTEE status (high confidence [HC] or low confidence [LC]).
+  - `major_consequence`: Primary consequence associated with transcript.
+  - `transcript_id`: Unique transcript ID.
+  - `transcript_version`: Transcript version.
+  - `gene_version`: Gene version.
+  - `is_mane_select`: Whether transcript is the MANE select transcript.
+  - `is_mane_select_version`: MANE Select version, has a value if this transcript is the MANE select transcript.
+  - `refseq_id`: RefSeq ID associated with transcript.
+  - `refseq_version`: RefSeq version.
+
+#### gnomAD v4.1. browser gene models Hail Table annotations
+
+Row fields:
+
+- `interval`: Struct representing start and end positions of gene.
+- `gene_id`: Unique gene ID.
+- `gene_version`: Gene version.
+- `gencode_symbol`: GENCODE gene symbol.
+- `chrom`: Chromosome in which gene is located.
+- `strand`:Gene strand.
+- `start`: Gene genomic start position (position only).
+- `stop`: Gene genomic stop position (position only.- `xstart`: Gene genomic start position (format: chromosomeposition).Note that chrX is encoded as 23, chrY as 24, and chrM as 25.
```

Comment:
```suggestion
- `stop`: Gene genomic stop position (position only).
- `xstart`: Gene genomic start position (format: chromosomeposition). Note that chrX is encoded as 23, chrY as 24, and chrM as 25.
```
this is pretty self explanatory, but it might be nice to have an example here as well